packages/gen-ai-hub/src/orchestration/orchestration-client.test.ts

```diff
@@ -28,92 +37,89 @@ describe('GenAiHubClient', () => {
     nock.cleanAll();
   });
 
-  it(' calls chatCompletion and parses response', async () => {
+  afterAll(() => {
+    clearXsuaaServices();
+  });
+
+  it('calls chatCompletion with minimum configuration and parses response', async () => {
     const module_configurations: ModuleConfigs = {
       templating_module_config: {
         template: [{ role: 'user', content: 'Hello!' }]
       },
```

Comment:
For prompt templating, `templating_module_config`, I would not ask the user to provide the complete object like the current status below, as it should be built by the SDK:
```
templating_module_config: {
         template: [{ role: 'user', content: 'Hello!' }]
       }
```

Instead, I would use the same API as our OpenAI client to keep similar UX.
Variables (next task in the prompt templating) can also be treated in the same way, as the only difference from the API perspective, would be additional information (input_param) to tell the value of the variables.

Reply:
As discussed, we will do this in a separate PR